src/BlogPostPage.jsx

```diff
@@ -0,0 +1,95 @@
+const GRAPHQL_ENDPOINT = props.GRAPHQL_ENDPOINT || "https://near-queryapi.api.pagoda.co";
```

Comment:
Here's the missing BlogPostPage :clap

Reply:
This PR adds it, but it doesn't appear to be in use anywhere. Is that intentional?

Reply:
I swapped out to use the BlogPostPage so that as we continue to build out we can leverage a different indexer, queries, etc if we so choose as well as the back button to the blog.